SPECS/expat/expat.spec

```diff
@@ -1,71 +1,77 @@
-Summary:	An XML parser library
-Name:		expat
-Version:	2.2.6
-Release:    4%{?dist}
-License:	MIT
-URL:		https://libexpat.github.io/
-Group:		System Environment/GeneralLibraries
+Summary:        An XML parser library
+Name:           expat
+Version:        2.4.1
+Release:        1%{?dist}
+License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:        https://github.com/libexpat/libexpat/releases/download/R_2_2_6/%{name}-%{version}.tar.bz2
-Patch0:         CVE-2018-20843.patch
-Requires:       expat-libs = %{version}-%{release}
+Group:          System Environment/GeneralLibraries
+URL:            https://libexpat.github.io/
+
+%define underscore_version $(echo %{version} | cut -d. -f1-3 --output-delimiter="_")
```

Comment:
Nit-pick: I'd move this macro to the top of the spec file for the sake of consistency - that's where we usually keep them and to not separate to common .spec tags. You might need to replace `%define` with `%global`, though - I think `%define` got interpreted on the line it was declared, so it wouldn't know about the value for `%{version}`.